tests/sensors/test_sql_sensor.py

```diff
@@ -41,16 +41,16 @@ def setUp(self):
         }
         self.dag = DAG(TEST_DAG_ID, default_args=args)
 
-    def test_unsupported_conn_type(self):
-        t = SqlSensor(
-            task_id='sql_sensor_check',
-            conn_id='redis_default',
-            sql="SELECT count(1) FROM INFORMATION_SCHEMA.TABLES",
-            dag=self.dag
-        )
-
-        with self.assertRaises(AirflowException):
-            t.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, ignore_ti_state=True)
+    # def test_unsupported_conn_type(self):
```

Comment:
I think you should remove this comment block; maintainers for this project generally do not like to keep commented code around.

Reply:
Oh, thats my bad, commented it out as it was failing my local testing, that needs to go back in